Makefile

```diff
@@ -51,14 +51,12 @@ clean-test: ## remove test and coverage artifacts
 typings/uvicorn/__init__.pyi:
 	pyright --createstub uvicorn
 
-check: typings/uvicorn/__init__.pyi ## type check with pyright
+check:
+	echo "Checking types with pyright"
+	$(MAKE) typings/uvicorn/__init__.pyi
```

Comment:
Why not keep this file as a dependency for the make target?

Reply:
Good point, will fix this!